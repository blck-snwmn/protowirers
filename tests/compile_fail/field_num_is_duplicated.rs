use protowirers::*;

#[derive(Proto)]
struct Sample {
    #[def(field_num = 1, field_num = 2, def_type = "int32")]
    s: u32,
}

fn main() {}
