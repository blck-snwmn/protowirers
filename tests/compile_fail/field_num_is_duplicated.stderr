error: field_num is duplicated in #[def(...)].
 --> $DIR/field_num_is_duplicated.rs:5:26
  |
5 |     #[def(field_num = 1, field_num = 2, def_type = "int32")]
  |                          ^^^^^^^^^^^^^
